apps/L1E/instagram/instagram-backend/src/resolvers/mutations/user/login.ts

```
@@ -0,0 +1,41 @@
+
+    const token = jwt.sign({ userId: user._id }, getJwtSecret());
+
+    return {
+      user,
+      token,
+    };
+  } catch (error) {
+    if (error instanceof Error) throw error;
+    throw new Error("Login failed");
+  }
+};
```

ESLint

[error] Async arrow function has a complexity of 6. Maximum allowed is 4.

(complexity)